singer-connectors/tap-postgres/requirements.txt

```diff
@@ -1 +1 @@
-pipelinewise-tap-postgres==1.6.3
+-e git+https://github.com/transferwise/pipelinewise-tap-postgres.git@AP-369#egg=pipelinewise-tap-postgres
```

Comment:
can you please bump ppw-tap-postgres to 1.6.4? It's now available on PyPI

Reply:
Sure